install/kubernetes/helm/istio/charts/pilot/templates/deployment.yaml

```diff
@@ -214,6 +218,11 @@ spec:
         secret:
           secretName: istio.istio-pilot-service-account
           optional: true
+{{- if .Values.jwksResolverExtraRootCA }}
+      - name: extracacerts
+        configMap:
+          name: istio-jwks-extra-cacerts
```

Comment:
why is this extra? shouldnt it be istio-jwks-root-ca ?

Reply:
Well, we include a bundle of standard CA certificates in the container, anything you add here will *also* be trusted, in addition to that included bundle.